homeassistant/components/rfxtrx.py

```diff
@@ -191,6 +214,78 @@ def get_rfx_object(packetid):
     return obj
 
 
+def get_pt2262_deviceid(device_id, nb_data_bits):
+    """Extract and return the address bits from a Lighting4/PT2262 packet."""
+    import binascii
+    try:
+        data = bytearray.fromhex(device_id)
+    except ValueError:
+        return None
+
+    mask = 0xFF & ~((1 << nb_data_bits) - 1)
+
+    data[len(data)-1] &= mask
+
+    return binascii.hexlify(data)
+
+
+def get_pt2262_cmd(device_id, data_bits):
+    """Extract and return the data bits from a Lighting4/PT2262 packet."""
+    try:
+        data = bytearray.fromhex(device_id)
+    except ValueError:
+        return None
+
+    mask = 0xFF & ((1 << data_bits) - 1)
+
+    return hex(data[-1] & mask)
+
+
+# pylint: disable=unused-variable
+def get_pt2262_device(device_id):
+    """Look for the device which id matches the given device_id parameter."""
+    for dev_id, device in RFX_DEVICES.items():
+        try:
```

Comment:
Check if the device is of type RfxtrxBinarySensor, then you do not need the "try"